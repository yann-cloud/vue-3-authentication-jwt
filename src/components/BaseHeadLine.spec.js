import {mount} from "@vue/test-utils";
// import {expect, test} from "vitest";
import BaseHeadLine from "./BaseHeadLine.vue";

const wrapper = mount(BaseHeadLine, {});

it('should render BaseHeadLine', () => {
    expect(wrapper.exists()).toBe(true);
})

it('should test snapshot UI testing', () => {
    expect(wrapper.text()).toMatchSnapshot();
})
