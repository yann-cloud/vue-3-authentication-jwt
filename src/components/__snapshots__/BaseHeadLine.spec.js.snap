// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should test snapshot UI testing 1`] = `""`;
